builder/vmware/common/driver_workstation_unix.go

```diff
@@ -116,26 +116,23 @@ func workstationNetmapConfPath() string {
 	return filepath.Join(base, "netmap.conf")
 }
 
-func workstationToolsIsoPath(flavor string) string {
-	return "/usr/lib/vmware/isoimages/" + flavor + ".iso"
+func workstationToolsIsoPath(appPath, flavor string) string {
+	libPath := filepath.Dir(filepath.Dir(appPath))
+	return filepath.Join(libPath, "vmware/isoimages/"+flavor+".iso")
 }
 
-func workstationVerifyVersion(version string) error {
+func workstationVerifyVersion(appPath, version string) error {
 	if runtime.GOOS != "linux" {
 		return fmt.Errorf("The VMware WS version %s driver is only supported on Linux, and Windows, at the moment. Your OS: %s", version, runtime.GOOS)
 	}
 
-	//TODO(pmyjavec) there is a better way to find this, how?
-	//the default will suffice for now.
-	vmxpath := "/usr/lib/vmware/bin/vmware-vmx"
-
-	var stderr bytes.Buffer
-	cmd := exec.Command(vmxpath, "-v")
-	cmd.Stderr = &stderr
+	var stdout bytes.Buffer
+	cmd := exec.Command(appPath, "-v")
```

Comment:
I think we're quite safe here, vmware executable is required by the [workstation driver](https://github.com/hashicorp/packer-plugin-vmware/blob/d4fc272/builder/vmware/common/driver_workstation9.go#L143) and if not found than it will not work at all. The `vmware` gives a bit different version than `vmware-vmx`, but at least the same:
* `vmware-vmx -v`
   ```
   $ /usr/local/lib/vmware/bin/vmware-vmx -v
   
   VMware Workstation Information:
   VMware Workstation 16.2.3 build-19376536 Release
   
   ```
* `vmware -v`:
   ```
   $ vmware -v
   VMware Workstation 16.2.3 build-19376536
   ```

Not sure how about the previous versions though... Right now the original (1.8.1) logic works for me with a workaround I described above.

Reply:
Actually for additional safety I can leave the old logic as plan B in case the execution of appPath or matching of version will not work - so everyone will be happy, wdyt?

Reply:
With the output you provide and the version logic (it only takes the major version), I'd think we're safe with the implementation you propose.
No need for a plan B I'd think, I was probably just a bit overcautious.